### DISCUSSION TITLE
make reindex more prominent

### DISCUSSION THREAD

**Discussion by danielb2** [namidaco/namida#78] [Ideas]
indexing initially was easy to add some content, but then after I added more albums, it felt way too tucked away. I think a reindex button in the Folders view next to settings makes a lot of sense.

**Comment by MSOB7YY** [comment]
hmm im not sure if this is a good place, the appbar can get cluttered based on the current page & running processes

i definitely want to make it easier to refresh library, just that i cant find any real good place, ppl may give good ideas here

btw, refreshing library (for differences) will be better than re-indexing

**Reply by danielb2** [reply]
I meant refreshing, sorry. Usually the term is rescan, so I'm getting it confused. 

OK, so in the Index settings page, I think a good start would be to move it to the top instead of having to scroll all the way down.  Realistically I imagine this would be the most used feature. Maybe you can implement some trcking to see how people use the app too, so you know what things people are clicking on the most

**Reply by MSOB7YY** [reply]
are u telling me to implement tracking in namida 💀
also putting re-index/refresh at the first place is DEFINITELY HARDER to access.. as long as u dont have alien fingers 💀

one possible thing is a top right icon in indexing page that refreshes the library, that would be a good start

**Reply by danielb2** [reply]
for me I have to scroll down to see  in the Indexer menu

**Reply by MSOB7YY** [reply]
reminder there is now a button for refreshing in indexer page card + pull to refresh in tracks page
